spec/controllers/sessions_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 
-require 'spec_helper' 
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.